packages/components/src/components/Box/Box.tsx

```diff
@@ -31,10 +32,10 @@ const mapVariantToBackgroundColor = ({ variant, theme }: MapArgs): CSSColor => {
     return theme[colorMap[variant]];
 };
 
-export interface BoxProps extends HTMLAttributes<HTMLDivElement> {
+export type BoxProps = FrameProps & {
```

Comment:
nit: someone was telling me that we shall use intefaces over types, I dont care but maybe we can open this in next code-improvements session